BrainPortal/app/controllers/userfiles_controller.rb

```diff
@@ -1333,16 +1333,17 @@ def export_file_list
     # Create the new file list
     file_list = CbrainFileList.new(
       :user_id          => current_user.id,
-      :group_id         => current_project.try(:id) || current_user.own_group.id,
+      :group_id         => if current_project.try(:public) then current_user.own_group.id else current_assignable_group_id end,
       :name             => "file_list.#{Process.pid}.#{Time.now.to_i}.cbcsv",
       :data_provider_id => dest_dp_id,
-    )
+      )
 
     # Save it and set its content.
     if file_list.save
       csv_text = CbrainFileList.create_csv_file_from_userfiles(userfiles)
       file_list.cache_writehandle { |fh| fh.write(csv_text) }
-      flash[:notice] = "Created file list named '#{file_list.name}'."
+      flash[:notice] = "Created file list named '#{file_list.name}' in '#{file_list.group.name}' project."\
```

Comment:
Why the backlash string continuation? It's very ugly. We never use this in CBRAIN

Reply:
ok